### PR TITLE
[Gdpr] Revert TCF v2 purpose consent logic

### PR DESCRIPTION
**Pull request by vguerci**
As we're considering to do this server side instead

### CODE REVIEW

File: CriteoPublisherSdk/Sources/DataConsent/CR_GdprVersion.m [hunk 1]
```diff
@@ -53,42 +41,26 @@ @implementation CR_GdprVersionWithKeys
...
                               appliesKey:(NSString *)appliesKey
                               tcfVersion:(CR_GdprTcfVersion)tcfVersion
                             userDefaults:(NSUserDefaults *)userDefaults {
   if (self = [super init]) {
-    _consentStringKey = [consentStringKey copy];
-    _purposeConsentsKey = [purposeConsentsKey copy];
-    _publisherRestrictionsKeyFormat = [publisherRestrictionsKeyFormat copy];
-    _vendorConsentsKey = [vendorConsentsKey copy];
-    _vendorLegitimateInterestsKey = [vendorLegitimateInterestsKey copy];
+    _consentStringKey = [constantStringKey copy];
```
Comment (TurpIF) [inline CriteoPublisherSdk/Sources/DataConsent/CR_GdprVersion.m:63]: what do you think about fixing this typo ?

**Review by TurpIF — APPROVED** [review]
-1K LoC, not bad 👍

File: CriteoPublisherSdk/Sources/DataConsent/CR_GdprVersion.m (hunk 1 above)
Comment (vguerci) [reply CriteoPublisherSdk/Sources/DataConsent/CR_GdprVersion.m:63]: Dang, thought I had not reverted that fix, will amend ¯\\\_(ツ)_/¯

**Review by TurpIF — APPROVED** [review]

**Comment by vguerci** [comment]
Thanks!

**Pull request merged**